src/components/StateEligibility.js

```diff
@@ -22,61 +24,124 @@ const useStyles = makeStyles((theme) => ({
-            ],
-            [
-                "https://www.mass.gov/info-details/covid-19-vaccinations-for-individuals-with-certain-medical-conditions",
-                "Individuals with two or more of certain medical conditions",
-            ],
-            [
-                "https://www.mass.gov/info-details/covid-19-vaccinations-for-senior-housing-settings",
-                "Residents and staff of low-income and affordable senior housing",
-            ],
-            [
-                "https://www.mass.gov/info-details/covid-19-vaccinations-for-people-ages-75-and-older",
-                "Individuals age 75 and older",
-            ],
-            [
-                "https://www.mass.gov/info-details/massachusetts-covid-19-vaccination-phases#phase-1-",
-                "People in Phase 1 (healthcare, nursing homes, etc.)",
-            ],
+            // NOTE: the link is applied to the section of text in the [square brackets]
+            {
+                startDate: "2021-03-11T00:00:00-05:00", // current timezone offset is at the end
+                link:
+                    "https://www.mass.gov/info-details/covid-19-vaccinations-for-k-12-educators-child-care-workers-and-school-staff",
+                text:
+                    "[K-12 educators, child care workers and K-12 school staff]",
+                color: "primary",
+            },
+            {
+                link:
+                    "https://www.mass.gov/info-details/covid-19-vaccinations-for-people-ages-65-and-older",
+                text: "Individuals [age 65 and older]",
+                color: "primary",
+            },
+            {
+                link:
+                    "https://www.mass.gov/info-details/covid-19-vaccinations-for-individuals-with-certain-medical-conditions",
+                text:
+                    "Individuals with [two or more of certain medical conditions]",
+                color: "primary",
+            },
+
+            {
+                link:
+                    "https://www.mass.gov/info-details/covid-19-vaccinations-for-senior-housing-settings",
+                text:
+                    "Residents and staff of [low-income and affordable senior housing]",
+                color: "primary",
+            },
+
+            {
+                link:
+                    "https://www.mass.gov/info-details/covid-19-vaccinations-for-people-ages-75-and-older",
+                text: "Individuals [age 75 and older]",
+                color: "primary",
+            },
+
+            {
+                link:
+                    "https://www.mass.gov/info-details/massachusetts-covid-19-vaccination-phases#phase-1-",
+                text: "People in [Phase 1] (healthcare, nursing homes, etc.)",
+                color: "primary",
+            },
         ],
     },
     /* TODO - remove the following div after March 11, and update link to be appropriate link from https://www.mass.gov/covid-19-vaccine */
```

Comment:
I love that it will automatically switch over at midnight. And then we can still remove the below code tmrw.